.travis.yml

```diff
@@ -17,9 +17,11 @@ matrix:
   include:
     - env: SUITE=test
     - env: SUITE=test
-      rust: "1.30.0"
+      rust: "1.31.0"
     - env: SUITE=format-check
+      rust: "1.31.0"
```

Comment:
FWIW, this would require us to run 1.31.0 locally now, too. It might be better to leave the minimum version only for tests.